sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -291,6 +292,39 @@ public CosmosClientBuilder keyCredential(CosmosKeyCredential cosmosKeyCredential
         return this;
     }
 
+    /**
+     * Gets the boolean which indicates whether to only return the headers and status code in Cosmos DB response
+     * in case of Create, Update and Delete operations on CosmosItem.
+     *
+     * If true, this removes the resource from response. It reduces networking
+     * and CPU load by not sending the resource back over the network and serializing it
+     * on the client.
+     *
+     * By-default, this is true.
+     *
+     * @return a boolean indicating whether resource will be included in the response or not
+     */
+    boolean isNoContentResponseOnWrite() {
+        return noContentResponseOnWrite;
+    }
+
+    /**
+     * Sets the boolean to only return the headers and status code in Cosmos DB response
+     * in case of Create, Update and Delete operations on CosmosItem.
+     *
+     * If set to true, this removes the resource from response. It reduces networking
+     * and CPU load by not sending the resource back over the network and serializing it on the client.
```

Comment:
NIT: I think pretty much all customers identifying this setting will also ask themselves whether this flag will have impact on RU charges. Would be good to document that it doesn't (at least my understanding) here - or at least in the FAQ for the feature.

Reply:
I like that idea, will document that "this doesn't impact RU usage for read or write operations"